docs/topics/migrations.txt

```diff
@@ -276,6 +276,14 @@ class to make it importable::
 Please refer to the notes about :ref:`historical-models` in migrations to see
 the implications that come along.
 
+.. _initial-property:
+
+.. note::
+
+    You may add an ``initial = True`` class property to the ``Migration`` class
+    to have Django apply the "initial migration" logic when the
+    :djadminopt:`--fake-initial` argument is used.
```

Comment:
I've addressed the 2nd addition in the comment above, I agree with the 1st addition, will do that..